deploy/crds/app.kiegroup.org_v1alpha1_kogitotrustyui_cr.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoTrustyUI
+metadata:
+  name: trustyui
+spec:
+  size: 1
```

Comment:
I think you should review this file. This one is the auto-generated by the SDK, right?